unittests/Evaluate/CMakeLists.txt

```diff
@@ -134,15 +134,6 @@ target_link_libraries(folding-test
 )
 
 set(FOLDING_TESTS
```

Comment:
The empty list should also be removed, as well as any references to it lower down the file.